route/route.go

```diff
@@ -132,6 +132,7 @@ func (r *Router) LnS(incomingOrPeer string) {
 	muxxer.HandleFunc("/alive", r.alive).Name("local health")
 	muxxer.HandleFunc("/panic", r.panic).Name("intentional panic")
 	muxxer.HandleFunc("/version", r.version).Name("report version info")
+	muxxer.HandleFunc("/getNode/{traceID}", r.getNode).Name("get node address for given trace ID")
```

Comment:
Preference for `/get-node/` or `/node/` or `/shard/` here, but that's NBD

Reply:
+1 

Case sensitivity in URL paths stands out to me so I'd prefer to ignore it. 

Also, thinking about future-proofing - what if we just had a debug endpoint that accepted a traceID? 

```
muxxer.HandleFunc("/debug/{traceID}", r.debug).Name("get debug information about a specific trace")
```

Then we're free to add more information to this as we go.

Reply:
Yeah, I think that's a good idea and agree having all lower cased paths is a good idea.

I think the path `/debug/trace/{traceID}` is more descriptive and more REST like incase we want to provide debug info on other types of data in the future.